src/views/TechnicalAssistance/AdminHome/InitialRequestForm.tsx

```diff
@@ -62,6 +63,14 @@ const InitialRequestForm = ({
     >
       {loading && <PageLoading />}
       {error && <NotFoundPartial />}
+
+      {/* Alert for  initial request form still in draft */}
```

Comment:
```suggestion
      {/* Alert for  initial request form still in progress or not yet started */}
```

Minor nit, but this could be the case, given the possible statuses

https://github.com/CMSgov/easi-app/blob/3be0b9673dc2d20b36932961a766c16e3dfc0096/src/types/graphql-global-types.ts#L232-L239